styles/main.css

```diff
@@ -146,11 +146,33 @@ img.h1 {
     display: block;
 }
 
-img.hero {
-    width: 55%;
-    min-width: 288px;
-    display: block;
-    margin: 0 auto;
+/******************
+* Responsive Hero *
+******************/
+.hero {
+    padding-top: 0;
+    height: 176px;
+    background: url(../images/responsive-images/notebook-small.png) 50% /contain no-repeat;
+}
+
+@media all and (min-width: 530px) {
+    .hero {
```

Comment:
I think things are so inconsistent with breakpoints right now that it's a separate but related issue.